src/main/kotlin/net/emdal/tank/Query.kt

```diff
@@ -1,10 +1,24 @@
 package net.emdal.tank
 
+import net.emdal.tank.Keyword.*
 import org.neo4j.driver.Session
 
+fun Session.query(block: Graph.() -> Graph) =
+  this.run("${Graph().block().query} RETURN *").list().toList()
+
+fun Graph.keyword(keyword: Keyword): Graph {
+  return this.copy(query = "$query\n${keyword.name} ".trimIndent())
+}
+
 fun Graph.match(graph: Graph.() -> Graph): Graph {
-  return this.copy(query = "$query\nMATCH ".trimIndent()).graph()
+  return this.keyword(MATCH).graph()
 }
 
-fun Session.query(block: Graph.() -> Graph) =
-  this.run("${Graph().block().query} RETURN *").list().toList()
\ No newline at end of file
+fun Graph.create(graph: Graph.() -> Graph): Graph {
+  return this.keyword(CREATE).graph()
+}
+
+enum class Keyword {
```

Comment:
I think we should use `clause` in place of `keyword` to be more in line with Cypher terminology. https://neo4j.com/docs/cypher-manual/current/clauses/

Reply:
We can leave this for a later PR. Wording and terminology will need brush ups continuosly